firestore/scripts/FriendlyEats.js

```diff
@@ -90,6 +109,23 @@ FriendlyEats.prototype.getCleanPath = function(dirtyPath) {
   }
 };
 
+FriendlyEats.prototype.updateUserInfo = function() {
+  var userID = firebase.auth().currentUser.uid;
+  var userData = {'lastLoginTime': Date()};
```

Comment:
No need for strings here and make sure to call `new`:

```ts
var userData = { lastLoginTime: new Date() };
```

Reply:
Fixed